src/pages/CreateProfile/Level.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from "react";
```

Comment:
Should: resolve errors in console

Warning: Each child in a list should have a unique "key" prop.

Reply:
should be fixed